source/recipes/troubleshooting.md

```diff
@@ -0,0 +1,40 @@
+# Troubleshooting
+
+## What to do if a binary cache is down or unreachable?
+
+Pass `--option substitute false` to Nix commands.
```

Comment:
```suggestion
Pass [`--option substitute false`](https://nixos.org/manual/nix/stable/command-ref/conf-file#conf-substitute) to Nix commands.
```

Pro tip: Whenever you have some command or term that is not defined in context, link to reference documentation. How are you going to find that? Well, how are readers going to find it without a link? Ask reviewers if you need help :)